python_modules/dagster/dagster/core/telemetry.py

```diff
@@ -42,6 +44,19 @@
 START_DAGIT_WEBSERVER = "start_dagit_webserver"
 DAEMON_ALIVE = "daemon_alive"
 TELEMETRY_VERSION = "0.2"
+DAGSTER_VERSION: List[
+    str
+] = []  # cache dagster version, but due to circular imports we must wait to import it.
+OS_DESC = platform.platform()
+
+
+def _get_dagster_version():
+    if not DAGSTER_VERSION:
+        from dagster import __version__ as dagster_version
+
+        DAGSTER_VERSION.append(dagster_version)
+    return DAGSTER_VERSION[0]
```

Comment:
i dont think you need to cache this if its behind a function - shouldn't be expensive